src/Simplex/Chat/View.hs

```diff
@@ -1871,6 +1908,7 @@ viewChatError logLevel testView = \case
     SEDuplicateGroupMessage {groupId, sharedMsgId}
       | testView -> ["duplicate group message, group id: " <> sShow groupId <> ", message id: " <> sShow sharedMsgId]
       | otherwise -> []
+    SENoteFolderNotFoundByName f -> ["no folder " <> ttyLocal f]
```

Comment:
+ check language in other places in view - to not confuse with file system, common to call directories as folders
```suggestion
    SENoteFolderNotFoundByName f -> ["no notes folder " <> ttyLocal f]
```